src/components/AppLayout/ListFilters/ListFilters.tsx

```diff
@@ -25,36 +27,45 @@ export const ListFilters = ({
   onFilterAttributeFocus,
   errorMessages,
   actions,
-}: ListFiltersProps) => (
-  <>
-    <Box
-      display="grid"
-      gridTemplateColumns={2}
-      gap={4}
-      paddingBottom={2}
-      paddingX={6}
-    >
-      <Box display="flex" alignItems="center" gap={4}>
-        <FiltersSelect
-          errorMessages={errorMessages}
-          menu={filterStructure}
-          currencySymbol={currencySymbol}
-          onFilterAdd={onFilterChange}
-          onFilterAttributeFocus={onFilterAttributeFocus}
-        />
+}: ListFiltersProps) => {
+  const isProductPage = window.location.pathname.includes("/products");
```

Comment:
**suggestion**: this looks wrong architecturally, I suppose filters are going to be used also in different views in the future? This should be a boolean prop I think

Reply:
Agree, generally ListFilters is implemented in quite wrong way, I can't now render different filters depending on the view, so just for feature flag i had to do this hack - otherwise, i have to refactor this component which i don't want to. Bu this "hack" i still have this within one place, additional prop would be temporary anyway